### PR TITLE
save a few bytes

### PR DESCRIPTION
**Pull request by yingjieMiao**
@eishay

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeeperInfo.scala [hunk 1]
```diff
@@ -54,7 +54,7 @@ object KeeperPageInfo {
     (__ \ 'keepersTotal).writeNullable[Int].contramap[Int](Some(_).filter(_ > 0)) and
     (__ \ 'libraries).writeNullable[Seq[JsObject]].contramap[Seq[JsObject]](Some(_).filter(_.nonEmpty)) and
     (__ \ 'keeps).write[Seq[KeepData]] and
-    (__ \ 'related).write[Seq[RelatedPageInfo]]
+    (__ \ 'related).write[Option[Seq[RelatedPageInfo]]]
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeeperInfo.scala:57]: Does this actually work? The fields above use `.writeNullable` and `.contramap` to omit them entirely when empty.

**Comment by yingjieMiao** [comment]
thanks, you are right. will change that.

**Comment by 2is10** [comment]
Okay, thanks for letting me know. I wasn’t sure. I thought maybe a recent Play release had made it easier to omit keys.

**Comment by yingjieMiao** [comment]
```
case class Foo(x: Int, y: Option[Seq[Int]])
implicit val fw = Json.writes[Foo]
```

This does omit keys. Apparently it needs more care when manually written.

**Pull request merged**